### ISSUE TITLE
pressKey('tab') on native input does not work properly on Firefox

### ISSUE THREAD

**Issue by Nikolaj1010** [DevExpress/testcafe#7342]
### What is your Scenario?

Use pressKey('tab') when native input is already focused by tab key to focus out

### What is the Current behavior?

When pressKey('tab') is used on already focused native input, focus remains in the same input

### What is the Expected behavior?

When pressKey('tab') is used on already focused native input, focus should jump from the native input to the next page element

### What is your public website URL? (or attach your complete example)

```
<!DOCTYPE html>
<html>
<body>

<input id="nativeInput" type="text"/>
<input id="nativeInput2" type="text"/>

<script>
    const inputEl = document.getElementById('nativeInput');
    inputEl.addEventListener('keypress', onKeyPress);

    function onKeyPress(event) {
        const eventValue = String.fromCharCode(event.keyCode);
        const isValidByPattern = /^[0-1]*$/.test(eventValue);
        if (!isValidByPattern) {
            event.preventDefault();
        }
        return isValidByPattern;
    }
</script>

</body>
</html>
```

### What is your TestCafe test code?

```
test('Should pressing tab key inside native input', async () => {
    const nameInput = Selector('#nativeInput');
    const nameInput2 = Selector('#nativeInput2');
    await t
        .click(nameInput)
        .expect(nameInput.focused).ok()
        .pressKey('tab')
        .expect(nameInput.focused).notOk()
        .expect(nameInput2.focused).ok();
});
```

### Steps to Reproduce

1. Click on the first native input to focus it
2. Press tab key
3. Check that focus remains in the same input instead of jumping to the next native input on page

### TestCafe version

1.18.4

### Node.js version

v14.18.0

### Command-line arguments

testcafe firefox *.html

### Browser name(s) and version(s)

Firefox 106.0.1

### Platform(s) and version(s)

Windows 10

**Comment by miherlosev** [comment]
Hi @Nikolaj1010,

Thank you for the shared example. I've reproduced the issue.

**Comment by OilerGera** [comment]
Hello,
.pressKey('tab') don't work on Spinner Input on Firefox 108.0 as well. 

Browser name(s) and version(s)
Firefox 108.0

Platform(s) and version(s)
Windows 10

**Comment by Artem-Babich** [comment]
Fixed in #7623
